e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/jackson/JacksonEMTest.kt

```diff
@@ -0,0 +1,49 @@
+package org.evomaster.e2etests.spring.openapi.v3.jackson
+
+import com.foo.rest.examples.spring.openapi.v3.jackson.JacksonController
+import org.evomaster.core.problem.rest.HttpVerb
+import org.evomaster.e2etests.spring.openapi.v3.SpringTestBase
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+
+class JacksonEMTest: SpringTestBase() {
+
+    companion object {
+        @BeforeAll
+        @JvmStatic
+        fun init() {
+            initClass(JacksonController())
+        }
+    }
+
+    @Test
+    fun testGenericReadValue() {
```

Comment:
might try to split `testGenericReadValue` and `testTypeReadValue` into two test suites (ie, two test files), see whether it still fails inGA